src/components/DisplayProductData.jsx

```diff
@@ -1,38 +1,81 @@
-import React, { Component } from 'react';
-import { getData } from '../modules/productData';
+import React, { Component } from "react";
+import { getData } from "../modules/productData";
 
 class DisplayProductData extends Component {
-	state = {
-		productData: []
-	}
+  state = {
+    productData: []
+  };
 
-	componentDidMount() {
-		this.getProductData()
-	}
+  componentDidMount() {
+    this.getProductData();
+  }
 
-	async getProductData() {
-		let result = await getData();
-		this.setState({ productData: result.data.products })
-	}
+  async getProductData() {
+    let result = await getData();
+    this.setState({ productData: result.data.products });
+  }
 
-	render() {
-		let dataIndex
-		if (Array.isArray(this.state.productData) && this.state.productData.length ) {
-			dataIndex = (
-				<div id="index">
-					{this.state.productData.map(item => {
-						return <div key={item.id}>{item.name}{item.description}{item.price}</div>
-					})}
-				</div>
-			)
-		} 
+  render() {
+    let entreeIndex, starterIndex, dessertIndex;
 
-		return (
-			<div>
-				{dataIndex}
-			</div>
-		)
-	}
-}
+    if (
+      Array.isArray(this.state.productData) &&
+      this.state.productData.length
+    ) {
+      starterIndex = (
+        <div id="starter-index">
+          {this.state.productData.map(item => {
+            if (item.category === "starter") {
+              return (
+                <div key={item.id}>
+                  <h4>{item.name}</h4>
+                  {item.description} {item.price}kr
+                </div>
+							);
+						}
+          })}
+        </div>
+			);
+			entreeIndex = (
+        <div id="entree-index">
+          {this.state.productData.map(item => {
+            if (item.category === "entree") {
+              return (
+                <div key={item.id}>
+                  <h4>{item.name}</h4>
+                  {item.description} {item.price}kr
+                </div>
+							);
+						}
+          })}
+        </div>
+			);
+			dessertIndex = (
+        <div id="dessert-index">
+          {this.state.productData.map(item => {
+            if (item.category === "dessert") {
+              return (
+                <div key={item.id}>
+                  <h4>{item.name}</h4>
+                  {item.description} {item.price}kr
+                </div>
+							);
+						}
+          })}
+        </div>
+			);
+    }
 
-export default DisplayProductData;
\ No newline at end of file
+    return (
+      <>
+        <h2>Starters</h2>
+        {starterIndex}
+				<h2>Entrees</h2>
+				{entreeIndex}
+				<h2>Dessert</h2>
+			  {dessertIndex}
+      </>
+    );
+  }
+}
+export default DisplayProductData;
```

Comment:
code formatting seems way off in this file. Hard to read. Please fix that before I do a final review.

Reply:
Updated code formatting in all the files that was needed.